### PR TITLE
Phase2-TB70 Add a simulation setup for radiation hardness studies of HGCal wafers

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add a simulation setup for radiation hardness studies of HGCal wafers

#### PR validation:

Done by dumping the geometry and tested using Fireworks

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/plugins/DDHGCalWaferPartialRotated.cc [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #include <vector>
 #include <sstream>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (civanch) [inline Geometry/HGCalCommonData/plugins/DDHGCalWaferPartialRotated.cc:20]: @bsunanda , please close debug printout

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
+geometry

**Comment by bsunanda** [comment]
@srimanob Please approve this PR

File: Geometry/HGCalCommonData/plugins/DDHGCalWaferPartialRotated.cc (hunk 1 above)
Comment (bsunanda) [reply Geometry/HGCalCommonData/plugins/DDHGCalWaferPartialRotated.cc]: Done

**Comment by bsunanda** [comment]
@srimanob Please approve this PR

**Comment by rappoccio** [comment]
+1

**Pull request merged**